### ISSUE TITLE
UI should expose entity ids

### ISSUE THREAD

**Issue by coverbeck** [dockstore/dockstore#3872]
**Is your feature request related to a problem? Please describe.**
If you want to use the Dockstore API against a workflow, tool, etc. that you are viewing in the UI, it's a mild pain; you have to open the browser developer tools to see what the ids are.

**Describe the solution you'd like**
The UI should surface ids somehow. Maybe by themselves, maybe even sample curls using the ids, or maybe links to the OpenAPI method with parameters filled in (is that last one possible?).

For example, if on a workflow, the workflow id should be visible somehow. The version ids should be visible on the versions table, etc.

**Describe alternatives you've considered**
Leave it as is. Maybe this is something only the Dockstore engineers need this and we shouldn't pollute the UI with it.

**Comment by denis-yuen** [comment]
Is this related to https://github.com/dockstore/dockstore/issues/3666 ?

**Comment by garyluu** [comment]
I think the difference is one's a TRS ID (with `#workflow/`) and one is a Dockstore ID

**Comment by denis-yuen** [comment]
The database ID?

**Comment by garyluu** [comment]
Yes, or whatever the ID something like https://dev.dockstore.net/api/static/swagger-ui/index.html#/containers/getContainer is using

**Comment by denis-yuen** [comment]
Yeah, then possibly some for curator/admin role in lieu of a developer role
